src/external/token/ERC20Issuance_blacklist_v1.sol

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+// Imports
+
+// Internal
+import {ERC20Issuance_v1} from "@ex/token/ERC20Issuance_v1.sol";
+import {IERC20Issuance_Blacklist_v1} 
+    from "@ex/token/interfaces/IERC20Issuance_Blacklist_v1.sol";
+
+// External
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+import {ERC20} from "@oz/token/ERC20/ERC20.sol";
+import {ERC20Capped} from "@oz/token/ERC20/extensions/ERC20Capped.sol";
+
+/**
+ * @title   ERC20 Issuance Token with Blacklist Functionality
```

Comment:
Please add the contract version and Inverter standard version to all of the contract and interface natspecs

See link https://github.com/InverterNetwork/standard?tab=readme-ov-file#file-structure